apps/web/src/ee/clerk/pages/ManageAccountPage.styles.ts

```diff
@@ -0,0 +1,94 @@
+import { css } from '@novu/novui/css';
+
+export const clerkComponentAppearance = {
+  elements: {
+    navbar: { display: 'none' },
+    navbarMobileMenuRow: { display: 'none !important' },
+    cardBox: {
+      display: 'block',
+      width: 'full',
+      height: 'full',
+      boxShadow: 'none',
+    },
+  },
+};
+
+export const modalStyles = {
+  modal: css({
+    borderRadius: '150 !important',
+    backgroundColor: 'surface.panel !important',
+    padding: '0 !important',
+    width: '[90%] !important',
+    height: 'full !important',
+  }),
+  body: css({ height: 'full !important' }),
+  header: css({ marginBottom: '0 !important' }),
+  close: css({
+    position: 'fixed !important',
+    right: '75 !important',
+    top: '[10px] !important',
+    zIndex: '[2] !important',
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Sorry, missed this one. Please use one of the values from the system. Perhaps `docked`?‏